templates/formatted_address.html

```diff
@@ -0,0 +1,3 @@
+<div class="address-details" data-content="{{ address_lines|slice:'1:'|join:'<br>' }}" data-html="true" title="{{ address_lines.0 }}">
```

Comment:
Do we actually use `data-*` for anything?

Reply:
I've checked all `.data(`, `.dataset.`, `attr('data-` `getAttribute(` calls in our JS and failed to find anything that would point to `content`, or `html`. I've also checked the `attr(data-` in CSS, but that didn't turn up anything.

Lastly, I've done some code archeology going back to commits from 2013 when the file was extracted from templates. From those it looks like back in times of the yore we've had the [`order_details.js`](https://github.com/mirumee/saleor/blob/a4dc0d16922814485cd99062cfb1873c36d0b26a/order/static/js/order_details.js) file that used those for popover feature long gone, so it looks like it's safe to delete.

Reply:
I think it was used for [Bootstrap popovers](https://getbootstrap.com/docs/4.0/components/popovers/).